dist/controllers/bets.js

```diff
@@ -20,14 +20,24 @@ async function getBets(_req, res) {
 exports.getBets = getBets;
 async function getBetsPendings(_req, res) {
     try {
-        const pendingbets = await index_1.default.bets.find({ status: betEnum_1.BetEnum.PENDING });
```

Comment:
I don't know why are you still using no conventions when write variables... this whole file has the same issue on your variables